examples/http_config.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
added here:

https://github.com/relayrides/dunlop/commit/36a934218d8544a76e4a38989d528c295a5a6c1e

As I mentioned there the formatting looks weird, but it exactly matches how my test setup was working.  We can easily verify it by manually adding an image to the S3 via the console and making it public once we have the halfshell server started.